<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-BPQODCE" failures="1" tests="1" name="PositiveLoginTest" time="15.034" errors="0" timestamp="2024-03-26T20:15:26 CET">
  <testcase classname="com.loginpagestest.PositiveLoginTests" name="logInTest" time="5.698">
    <failure type="java.lang.AssertionError" message="expected [http://the-internet.herokuapp.com/secure] but found [https://the-internet.herokuapp.com/secure]">
      <![CDATA[java.lang.AssertionError: expected [http://the-internet.herokuapp.com/secure] but found [https://the-internet.herokuapp.com/secure]
at com.loginpagestest.PositiveLoginTests.logInTest(PositiveLoginTests.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- logInTest -->
</testsuite> <!-- PositiveLoginTest -->
